project-base/src/Shopsys/ShopBundle/Resources/styles/admin/todo.less

```diff
@@ -0,0 +1 @@
+
```

Comment:
Can you please add a comment here that says what is this file for? (it could be the same as the one in `main.less`

Reply:
added note to file todo.less and changed update notes.